.github/ISSUE_TEMPLATE/oss_contribution.yml

```
@@ -0,0 +1,57 @@
+name: OSS Contribution
+description: Add or Modify OSS Information to OSORI
+labels: ["oss", "review required"]
```

```suggestion
labels: ["data contribution", "review required"]
```
What about modifying the title of "oss" lable to "data contribution"?

---

Done